docs/maven-example.md

```diff
@@ -0,0 +1,88 @@
+# Fabric8 Maven Plugin + Kompose:
+Lets deploy a Springboot Java application with Docker Compose file using Fabric8 Maven Plugin to Kubernetes or OpenShift.
+
+##### Requirements
+* Linux or MacOS or Windows
+* JDK 1.7+ - [JDK Quick Installation Guide](http://openjdk.java.net/install/)
+* Maven 3.x+ - [Maven Installation Guide](http://www.baeldung.com/install-maven-on-windows-linux-mac) 
+* Kompose - [Kompose Installation Guide](/docs/installation)
+
+__1. Clone the example project from GitHub__
+```bash
+$ git clone https://github.com/piyush1594/kompose-maven-example.git
+```
+
+Change current directory to `kompose-maven-example` directory. Use command
+```bash
+$ cd kompose-maven-example
+```
+
+__2. Add Fabric8 Maven Plugin to your project__     
+```
+$ mvn io.fabric8:fabric8-maven-plugin:3.5.28:setup
+```
+
+Adds the Fabric8 Maven Plugin configuration to `pom.xml` of project. `pom.xml` is manifest or deployment descriptor file of a maven project.
+
+__3. Install Kompose through Maven__
+```
+$ mvn fabric8:install
+```
+
+This command installs the `kompose` on the host.
+
+__4. Configure Fabric8 Maven Plugin to use a Docker Compose file__
+```
+<plugin>
+  <groupId>io.fabric8</groupId>
+  <artifactId>fabric8-maven-plugin</artifactId>
+  <configuration>
+    <composeFile>path for docker compose file</composeFile>
+  </configuration>
+  <executions>
+    <execution>
+      <goals>
+        <goal>resource</goal>
+        <goal>build</goal>
+      </goals>
+    </execution>
+  </executions>
+</plugin>
+```
+
+Add the `<configuration>` and `<executions>` sections to `pom.xml` as shown in above `pom.xml` snippet. Update the `<composeFile>` to provide the relative path of Docker Compose file from `pom.xml`
+
+__5. Deploy application on Kubernetes or OpenShift__
```

Comment:
add blank line here

Reply:
@piyush1594 please fix this comment then LGTM.

Reply:
@cdrage I already fixed it in the last commit.